dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -253,7 +253,7 @@ public async Task<Plan> InvokeNextStepAsync(SKContext context)
             if (result.ErrorOccurred)
             {
                 throw new KernelException(KernelException.ErrorCodes.FunctionInvokeError,
-                    $"Error occurred while running plan step: {context.LastErrorDescription}", context.LastException);
+                    $"Error occurred while running plan step: {functionContext.LastErrorDescription}", functionContext.LastException);
```

Comment:
```suggestion
                    $"Error occurred while running plan step: {result.LastErrorDescription}", result.LastException);
```

This might be even more appropriate. But functionally equivalent to your change. Thanks!